pkg/skr/iprange/createKcpIpRange.go

```diff
@@ -45,7 +47,11 @@ func createKcpIpRange(ctx context.Context, st composed.State) (error, context.Co
 			Cidr: state.ObjAsIpRange().Spec.Cidr,
 		},
 	}
-
+	if _, exists := state.ObjAsIpRange().Annotations[string(cloudresourcesv1beta1.IpRangeCloudResources)]; exists {
```

Comment:
When would a user created SKR IpRange have this annotation? Think that for Azure we always need to create KCP IpRange with CM Network reference. For all other cases it still should be as before the KCP Kyma Network